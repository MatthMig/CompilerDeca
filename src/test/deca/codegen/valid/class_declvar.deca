// Description :
// 
//      Test correct du lexing d'une classe avec une declaration de variable
// 
// Résultats : 
// 
//      N/A 
// 
// Historique : 
// 
//      Créé le 19/06/2023 par gl03

class A {
    int x;
}
{
    println("La variable x est déclarée");
}
