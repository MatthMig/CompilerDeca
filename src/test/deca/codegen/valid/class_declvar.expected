La variable x est déclarée